services/api-server/src/simcore_service_api_server/api/routes/health.py

```diff
@@ -21,10 +25,21 @@ async def check_service_health():
 async def get_service_state(
     catalog_client: CatalogApi = Depends(get_api_client(CatalogApi)),
     director2_api: DirectorV2Api = Depends(get_api_client(DirectorV2Api)),
+    storage_client: StorageApi = Depends(get_api_client(StorageApi)),
+    url_for: Callable = Depends(get_reverse_url_mapper),
 ):
-    apis = (catalog_client, director2_api)
-    heaths: List[bool] = await asyncio.gather(*[api.is_responsive() for api in apis])
-
-    return {
-        api.service_name: {"healthy": is_healty} for api, is_healty in zip(apis, heaths)
-    }
+    apis = (catalog_client, director2_api, storage_client)
```

Comment:
this is actually used to check the state of the services connected to the ``api-server`` in the backend. 
NOTE that even if it is in ``health.py``, it is NOT the health-check entrypoint

For the moment we are using this to **manually** diagnose that everything is connected in the backend to the service correctly. E.g. we had a bug in the api-server was not able to connect to the director2 in staging because of a wrong environment variable added by operations  ... it simply by hitting /status entry point that we could see that

We can eventually connect this to graphana or promethous at some point.